econet_hvac_odu.yaml

```diff
@@ -34,6 +34,26 @@ sensor:
     request_mod: none
     accuracy_decimals: 1
     entity_category: "diagnostic"
+  - platform: econet
+    name: "ODU Pressure Liquid"
+    id: odu_pressure_liquid
+    sensor_datapoint: PRES_LIQ
+    request_mod: none
+    unit_of_measurement: "psi"
+    accuracy_decimals: 1
+    device_class: "pressure"
+    state_class: "measurement"
+    entity_category: "diagnostic"
+  - platform: econet
+    name: "ODU Liquid Line Temp"
+    id: odu_liquid_line_temp
+    sensor_datapoint: TEMP_OLT
+    request_mod: none
+    unit_of_measurement: "°F"
+    accuracy_decimals: 1
+    device_class: "temperature"
+    state_class: "measurement"
+    entity_category: "diagnostic"
```

Comment:
Given that these sensors aren't available on all ODU's, I suggest we add `disabled_by_default: true` to these sensors.

Reply:
do i put that for every sensor?

Reply:
Please for the ones you are adding related to suction, pressure, etc...as it seems they are not present on all ODU's.

Reply:
ok i made the updated to the two entries I added, hope I did it right, if not let me know!